features/environment.py

```diff
@@ -128,13 +129,18 @@ def _make_patroni_test_config(self, name, dcs, tags):
             config['tags'] = tags
 
         if dcs != 'etcd':
-            etcd = config.pop('etcd')
-            config['zookeeper'] = {'scope': etcd['scope'], 'session_timeout': etcd['ttl'],
-                                   'reconnect_timeout': config['loop_wait']}
-            if dcs == 'exhibitor':
-                config['zookeeper']['exhibitor'] = {'hosts': ['127.0.0.1'], 'port': 8181}
+            dcs_config = config.pop('etcd')
```

Comment:
No, for the behave tests we are parsing the configuration file manually, so instead of:

    dcs_config = config.pop('etcd')
    # do whatever it takes to make a zookeeper configuration out of it
    config['zookeeper'] = dcs_config
 
we would do

    config.pop('etcd')
    config.pop('consul')

leaving only zookeeper config in place.

I think the way we are currently doing this is quite obscure, i.e. I spent some time trying to figure out why do we pop the etcd config for either consul or zookeeper.